src/app/tests/suites/OTA_SuccessfulTransfer.yaml

```diff
@@ -0,0 +1,177 @@
+              - name: "discriminator"
+                value: providerDiscriminator
+              - name: "port"
+                value: providerPort
+              - name: "kvs"
+                value: providerKvs
+              - name: "filepath"
+                value: otaImageFilePath
+
+    - label: "Commission the provider from alpha"
+      identity: "alpha"
+      cluster: "CommissionerCommands"
+      command: "PairWithQRCode"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: providerNodeId
+              - name: "payload"
+                value: providerPayload
+
+    - label: "Wait for the commissioned provider to be retrieved for alpha"
+      identity: "alpha"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: providerNodeId
+
+    - label: "Install ACL for QueryImage"
+      cluster: "Access Control"
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value: [
+                  # Grant administer privilege to the default controller node ID 112233
+                  {
+                      FabricIndex: 1,
+                      Privilege: 5,
+                      AuthMode: 2,
+                      Subjects: [112233],
+                      Targets: null,
+                  },
+                  # Grant operate privileges to all nodes for the OTA Provider cluster on every endpoint
+                  {
+                      FabricIndex: 1,
+                      Privilege: 3,
+                      AuthMode: 2,
+                      Subjects: null,
+                      Targets: null,
+                      [{ Cluster: 41, Endpoint: null, DeviceType: null }],
+                  },
+              ]
+
+    - label: "Stop the requestor"
+      cluster: "SystemCommands"
+      command: "Stop"
+
+    - label: "Start the requestor with an OTA download path"
+      cluster: "SystemCommands"
+      command: "Start"
+      arguments:
+          values:
+              - name: "otaDownloadPath"
+                value: downloadImageFilePath
+
+    - label: "Wait for the commissioned requestor to be retrieved for alpha"
+      identity: "alpha"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: requestorNodeId
+
+    - label: "Send an announce OTA provider command to the requestor"
+      cluster: "OTA Software Update Requestor"
+      command: "AnnounceOtaProvider"
+      arguments:
+          values:
+              - name: "providerNodeId"
+                value: providerNodeId
+              - name: "vendorId"
+                value: 0
+              - name: "announcementReason"
+                value: 0
+              - name: "endpoint"
+                value: endpoint
+
+    - label: "Wait for transfer complete message"
```

Comment:
Where does this message come from? From logs?

Reply:
Yes, from logs. Failure of download would result in this message never being seen so it would timeout, causing the test to fail.